macros/src/main/scala/MacroCompiler.scala

```diff
@@ -121,15 +121,15 @@ class MacroCompilerPass(mems: Option[Seq[Macro]],
       for ((memPort, libPort) <- pairedPorts) {
 
         // Make sure we don't have a maskGran larger than the width of the memory.
-        assert (memPort.src.effectiveMaskGran <= memPort.src.width)
-        assert (libPort.src.effectiveMaskGran <= libPort.src.width)
+        assert (memPort.src.effectiveMaskGran <= memPort.src.width.get)
```

Comment:
In this context, yes they will all be Some, because the ports are not on a SRAMCompiler. Its a bit of a hack, but then again, making width an Option[Int] does sort of make sense.